consult.gemspec

```diff
@@ -27,12 +27,12 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'diplomat', '~> 2.0.2'
   spec.add_dependency 'vault', '>= 0.10.0', '< 1.0.0'
 
-  spec.add_development_dependency 'bundler', '~> 1.16'
```

Comment:
In the past I have run into issues between different versions of bundler. I'd be curious if everything works as expected between versions 1.16 and 2.2.

Reply:
Same. We can keep an eye on it.